core/include/detray/core/detector.hpp

```diff
@@ -502,6 +502,14 @@ class detector {
         _volume_finder = std::move(v_grid);
     }
 
+    /// Add the volume grid - copy semantics
+    ///
+    /// @param v_grid the volume grid to be added
+    DETRAY_HOST
+    inline auto add_volume_finder(const volume_finder &v_grid) -> void {
```

Comment:
Shouldn't it be `set_volume_finder` instead?

Reply:
Renamed